evalml/utils/logger.py

```diff
@@ -1,10 +1,6 @@
 import logging
-import os
 import sys
 import time
-import traceback
-from logging.handlers import RotatingFileHandler
-from pathlib import Path
 
 
 def get_logger(name):
```

Comment:
Now that we're not using this `get_logger` method in our source code, do we have unit test coverage for it?

Reply:
Yep, tests still exist in `get_logger`, just without the tests for file configuration!

We do still use this `get_logger` method when in verbose mode for AutoMLSearch and when describing pipelines and components, with associated tests.